centos_build/README.md

```diff
@@ -0,0 +1,47 @@
+## How it works?  
```

Comment:
You should add a initial section saying that this is centos build, and it is not officially supported, etc. You can just point to the section in the top-level readme that describes.